package.json

```diff
@@ -10,13 +10,15 @@
   "scripts": {
     "build": "webpack --config webpack.config.js",
     "build:proto": "protoc --js_out=import_style=commonjs:. --grpc-web_out=import_style=commonjs+dts,mode=grpcwebtext:. ./src/api/yorkie.proto",
-    "build:docs": "typedoc --includeVersion --ignoreCompilerErrors --excludeExternals --out docs src",
+    "build:docs": "api-documenter markdown --input temp --output docs",
     "start": "webpack-dev-server --watch --config webpack.dev.config.js",
     "test": "karma start karma.conf.js --single-run",
     "test:yorkie.dev": "karma start karma.conf.js --single-run --testRPCAddr=https://yorkie.dev/api",
     "test:watch": "karma start karma.conf.js",
     "lint": "eslint . --fix --max-warnings=0 --ext .ts",
-    "prepare": "npm run build"
+    "prepare": "npm run build",
+    "api-report": "npm run build && yarn api-report:api-json",
```

Comment:
```suggestion
    "api-report": "npm run build && npm run api-report:api-json",
```

Reply:
When I test this branch, I have to add api-extractor by running the script below.

```
npm install --save-dev @microsoft/api-extractor
```